app/src/main/scala/org/alephium/explorer/api/TokensEndpoints.scala

```diff
@@ -29,16 +29,25 @@ import org.alephium.protocol.model.{Address, TokenId}
 // scalastyle:off magic.number
 trait TokensEndpoints extends BaseEndpoint with QueryParams {
 
+  private lazy val maxTokens: Int = 80
+
   private val tokensEndpoint =
     baseEndpoint
       .tag("Tokens")
       .in("tokens")
 
-  val listTokens: BaseEndpoint[Pagination, ArraySeq[TokenId]] =
+  val listTokens: BaseEndpoint[(Pagination, Option[StdInterfaceId]), ArraySeq[TokenInfo]] =
     tokensEndpoint.get
       .in(pagination)
-      .out(jsonBody[ArraySeq[TokenId]])
-      .description("List tokens")
+      .in(tokenInterfaceIdQuery)
+      .out(jsonBody[ArraySeq[TokenInfo]])
+      .description("List token information")
```

Comment:
Do we still want to have an endpoint to list all tokens regardless of the std interface id? Im not sure there is a real use case for now, but maybe there will be

Reply:
yes, `interface-id` is an optional query param, if it's not provided we return everything.